src/commands/sentinel-debug.json

```diff
@@ -12,6 +12,73 @@
             "SENTINEL",
             "ONLY_SENTINEL"
         ],
+        "reply_schema": {
+            "oneOf": [
+                {
+                    "description": "The configuration update was successful.",
+                    "const": "OK"
+                },
+                {
+                    "description": "List of configurable time parameters and their values (milliseconds).",
+                    "type": "object",
+                    "additionalProperties": false,
```

Comment:
for this part, I think we do not need list all the parameters, we just need to write as help json file format is enough